hw/dv/data/tests/mem_tests.hjson

```diff
@@ -3,14 +3,23 @@
     {
       name: mem_tests_mode
       uvm_test_seq: "{name}_common_vseq"
-      run_opts: ["+csr_mem_walk", "+en_scb=0"]
+      run_opts: ["+en_scb=0"]
     }
   ]
 
   tests: [
     {
       name: "{name}_mem_walk"
       en_run_modes: ["mem_tests_mode"]
+      run_opts: ["+csr_mem_walk"]
```

Comment:
Hjson limits only one key-value pair. it will takes the last one. Try running the mem_walk test - it will likely say "test not found". 

Please have a single list of these dicts.

Reply:
Thanks, Sri. I accidentally copied 2 more lines. Have fixed it.